app/src/main/java/com/example/weatherapp/data/remote/ApiService.kt

```diff
@@ -15,14 +16,14 @@ interface ApiService {
     @GET("weather")
     suspend fun getCurrentCityWeather(
         @Query("q") cityName: String?,
-        @Query("units") units: String,
+        @Query("units") units: String = UnitSystem.STANDARD.value,
```

Comment:
Add the parameter value as constant in this file to avoid errors in the future if you decide to use `UnitSystem.value` for something